cmd/checkup-launcher.go

```diff
@@ -110,3 +124,11 @@ func logCheckupSpec(spec *checkup.Spec) {
 		log.Printf("\t\t%q\n", role)
 	}
 }
+
+func updateConfigMap(clientset *kubernetes.Clientset, namespace string, cm *corev1.ConfigMap) error {
+	if _, err := clientset.CoreV1().ConfigMaps(namespace).Update(context.Background(), cm, metav1.UpdateOptions{}); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
I think this can be moved to be a function that gets `workspace` as a receiver similar to `RetriveStatus`, just for updating the status.

Reply:
This ConfigMap object is the external one (not the one in the checkup workspace)